### DISCUSSION TITLE
拜托了

### DISCUSSION THREAD

**Discussion by cht19991018** [vercel/vercel#4430] [Help] [answered]
为什么我绑定GitHub的时候，提醒我您的电子邮件地址无效。给我们发电子邮件support@vercel.com寻求帮助。

**Comment by styfle** [comment, marked as answer]
Hi @cht19991018 

Are you unable to email support@vercel.com? Is there an error message?

You could also try https://vercel.com/support/request

**Comment by paulogdm** [comment]
Closing. Please reach out to our support channel for further information.
